cli/azd/internal/telemetry/resource/machine_id.go

```diff
@@ -67,10 +67,15 @@ func getMacAddress() (string, bool) {
 	for _, ift := range interfaces {
 		if len(ift.HardwareAddr) > 0 && ift.Flags&net.FlagLoopback == 0 {
 			hwAddr, err := net.ParseMAC(ift.HardwareAddr.String())
+			ipAddr, _ := ift.Addrs()
```

Comment:
NIT: could we move this check down 3 lines (so that it doesn't interleave between the call to ParseMAC and the place where we check the `err` return from `ParseMAC`)?